lib/rubocop/cop/style/empty_string_inside_interpolation.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for empty strings being assigned inside string interpolation.
+      #
+      # Empty strings are a meaningless outcome inside of string interpolation, so we remove them.
+      #
+      # While this cop would also apply to variables that are only going to be used as strings,
+      # RuboCop can't detect that, so we only check inside of string interpolation.
+      #
+      # @example
+      #   # bad
+      #   "#{condition ? 'foo' : ''}"
+      #
+      #   # good
+      #   "#{'foo' if condition}"
+      #
+      #   # bad
+      #   "#{condition ? '' : 'foo'}"
+      #
+      #   # good
+      #   "#{'foo' unless condition}"
+      #
+      class EmptyStringInsideInterpolation < Base
+        include Interpolation
+        extend AutoCorrector
+        MSG = 'Do not assign empty strings to variables inside string interpolation.'
```

Comment:
I find the mention of "variables" here confusing.

Reply:
Would `Do not assign empty strings inside string interpolation.` be a better way of phrasing it?

Reply:
Fixed.